tools/parse_module_type.py

```diff
@@ -224,6 +252,15 @@ def prequisitesCheck():
         return False
     return True
```

Comment:
This return is too much, remove please.

Reply:
The `return True`? That's your code. I can do, but you sure? If so, should I also delete line 262 then?